projects/discussion-ui/src/lib/components/discuss-home/discuss-home.component.ts

```diff
@@ -49,6 +49,14 @@ export class DiscussHomeComponent implements OnInit {
       this.getDiscussionList(_.get(this.routeParams, 'slug'));
     });
   }
+  /**
+   * @description - set the scroll container height
+   */
+  ngAfterViewChecked() {
+    if (this.elementView && this.elementView.nativeElement) {
```

Comment:
Good to have a line comment here what are we doing here & why did we used "ngAfterViewChecked" method